regexes/device/mobiles.yml

```diff
@@ -29180,6 +29218,34 @@ X-AGE:
   device: 'smartphone'
   model: 'G1 Lite'
 
+# Zonko
+Zonko:
+  regex: 'K101_EEA(?:[);/ ]|$)'
+  device: 'tablet'
+  model: 'K101'
+
+# TJC (www.tjc-jp.com)
+TJC:
+  regex: 'Metal_Tablet_10(?:[);/ ]|$)'
+  device: 'tablet'
+  model: 'Metal Tablet 10'
+
+# HOTREALS
+HOTREALS:
+  regex: 'HOTREALS'
+  device: 'tablet'
+  models:
+    - regex: 'HOTREALS(?:[);/]|$)'
```

Comment:
Does this device not have any other details in the useragent? Wondering if other devices of that brand might cause a incorrect detection then.

Reply:
At the moment. This is only device they have.